src/devices/bosch.ts

```diff
@@ -174,6 +182,26 @@ Example: 30ff00000102010001`;
 
 
 const tzLocal = {
+    broadcast_alarm: {
+        key: ['broadcast_alarm'],
+        convertSet: async (entity, key, value, meta) => {
+            if (key === 'broadcast_alarm') {
+                const index = utils.getFromLookup(value, broadcastAlarmState);
+                const broadcastFrame = Zcl.ZclFrame.create(
+                    Zcl.FrameType.SPECIFIC,
+                    Zcl.Direction.CLIENT_TO_SERVER,
+                    true,
+                    Zcl.ManufacturerCode.ROBERT_BOSCH_GMBH,
+                    71, // zclTransactionSequenceNumber
+                    'boschSmokeDetectorSiren',
+                    1280, // ssIasZone
+                    {data: index});
+                // @ts-expect-error
+                await meta.device.constructor.adapter.sendZclFrameToAll(255, broadcastFrame, 1, true);
```

Comment:
I know I proposed to do it like this earlier, but thinking about it again it would be cleaner to add a `broadcastCommand` to [`controller.ts`](https://github.com/Koenkk/zigbee-herdsman/blob/master/src/controller/controller.ts) and use it here (we will modify `meta` to pass the `zh` `controller`)

Reply:
If you have the time, please make a PR for it, maybe as part of https://github.com/Koenkk/zigbee-herdsman/pull/1028?